api/src/main/java/uk/gov/dhsc/htbhf/claimant/message/processor/CompletePaymentMessageProcessor.java

```diff
@@ -0,0 +1,43 @@
+package uk.gov.dhsc.htbhf.claimant.message.processor;
+
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+import uk.gov.dhsc.htbhf.claimant.entity.Message;
+import uk.gov.dhsc.htbhf.claimant.message.MessageStatus;
+import uk.gov.dhsc.htbhf.claimant.message.MessageType;
+import uk.gov.dhsc.htbhf.claimant.message.MessageTypeProcessor;
+import uk.gov.dhsc.htbhf.logging.event.FailureEvent;
+
+import javax.transaction.Transactional;
+
+import static uk.gov.dhsc.htbhf.claimant.message.MessageStatus.COMPLETED;
+import static uk.gov.dhsc.htbhf.claimant.message.MessageType.COMPLETE_PAYMENT;
+
+/**
+ * Processes COMPLETE_PAYMENT messages by calling the PaymentService.
+ */
+@Component
+@AllArgsConstructor
+@Slf4j
+public class CompletePaymentMessageProcessor implements MessageTypeProcessor {
+
+    @Override
+    @Transactional(Transactional.TxType.REQUIRES_NEW)
+    public MessageStatus processMessage(Message message) {
+
```

Comment:
shouldn't this be processing the message?

Reply:
Or have a TODO statement, at least

Reply:
Added TODO's